src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [search, setSearch] = useState('');
+
+  const visiableMovies = moviesFromServer.filter(film => (
+    film.title
+      .toLowerCase()
+      .split(search.toLowerCase().trim())
+      .length > 1
+    || film.description
+      .toLowerCase()
+      .split(search.toLowerCase().trim())
+      .length > 1
+  ));
```

Comment:
nope, move `earch.toLowerCase().trim()` from filter
Also, you can combine `film.title` and `film.description` in one string and use includes instead of `length > 1`